torch/onnx/symbolic_helper.py

```diff
@@ -408,18 +408,35 @@ def _interpolate_warning(interpolate_mode):
                   "We recommend using opset 11 and above for models using this operator.")
 
 def _unsqueeze_helper(g, input, axes_i):
-    if _export_onnx_opset_version >= 13:
-        axes = g.op("Constant", value_t=torch.tensor(axes_i, dtype=torch.long))
-        return g.op("Unsqueeze", input, axes)
-    else:
-        return g.op("Unsqueeze", input, axes_i=axes_i)
+    if _is_constant(axes_i[0]):
+        if _export_onnx_opset_version >= 13:
+            axes = g.op("Constant", value_t=torch.tensor(axes_i, dtype=torch.long))
+            return g.op("Unsqueeze", input, axes)
+        else:
+            return g.op("Unsqueeze", input, axes_i=axes_i)
+    else:  # Tensor type
+        if _export_onnx_opset_version >= 13:
+            axes_t = axes_i[0]
+            return g.op("Unsqueeze", input, axes_t)
+        else:
+            raise ValueError("Unsupported Unsqueeze dynamic axes ONNX opset version: " + str(_export_onnx_opset_version))
 
 def _squeeze_helper(g, input, axes_i):
-    if _export_onnx_opset_version >= 13:
-        axes = g.op("Constant", value_t=torch.tensor(axes_i, dtype=torch.long))
-        return g.op("Squeeze", input, axes)
-    else:
-        return g.op("Squeeze", input, axes_i=axes_i)
+    if _is_constant(axes_i[0]):
+        if _export_onnx_opset_version >= 13:
+            axes = g.op("Constant", value_t=torch.tensor(axes_i, dtype=torch.long))
+            return g.op("Squeeze", input, axes)
+        else:
+            return g.op("Squeeze", input, axes_i=axes_i)
+    else:  # Tensor type
+        if _export_onnx_opset_version >= 13:
+            axes_t = axes_i[0]
+            if _get_tensor_sizes(axes_t) and _get_tensor_sizes(axes_t) == 1:
```

Comment:
`_get_tensor_sizes(axes_t) == 1` this is incorrect. probably should be `len(_get_tensor_sizes(axes_t)) == 1`

Reply:
change it to `_get_tensor_rank`.